src/cloud_common/schemas/service_config/aos_config.py

```diff
@@ -0,0 +1,167 @@
+#
+#  Copyright (c) 2018-2024 Renesas Inc.
+#  Copyright (c) 2018-2024 EPAM Systems Inc.
+#
+from datetime import datetime, timedelta
+from typing import Annotated, Optional, Literal, Dict
+
+from pydantic import BaseModel, Field
+
+from cloud_common.schemas.service_config.aso_types import UnitDevice, RunParameters, ServiceQuotas, AlertRules
+
+
+class AosConfigSchema(BaseModel):
+    """
+    Aos service config schema.
+
+
+    """
+
+    created: Annotated[
+        datetime,
+        Field(
+            alias='created',
+            description='Timestamp when Aos service was created.',
+        ),
+    ]
+
+    author: Annotated[
+        Optional[str],
+        Field(
+            alias='author',
+            default=None,
+            description='Aos service author.',
+        ),
+    ]
+
+    balancing_type: Annotated[
+        Literal[
+            'enabled',
+            'disabled',
+        ],
+        Field(
+            alias='balancingType',
+            default='enabled',
+            description='Balancing type. `disabled` means total prohibition from balancing to other nodes.`',
+        ),
+    ]
+
+    service_provider: Annotated[
```

Comment:
This is new field. Should core handle it somehow? It is the same as received in desired message for service info?